### PR TITLE
Remove escalus warnings in privacy_SUITE

### PR DESCRIPTION
**Pull request by arkgil**
This PR removes warnings emitted by escalus when running privacy_SUITE.

Since [this commit](https://github.com/esl/escalus/commit/93ee6eef1cce80ae7d1213c62b31cfebe301b11f) escalus prints warning whenever we try to lookup JID by atom representing user in config file.

Proposed changes include:
* removed sending stanzas to recipients defined by atoms in `privacy_SUITE`
* removed setting privacy lists' targets  as atoms in `privacy_helpers` module

### CODE REVIEW

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 1]
```diff
@@ -186,18 +179,14 @@ list_content(<<"deny_localhost_iq">>) -> [
     ];
 list_content(<<"deny_group_iq">>) -> [
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"iq">>])
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/privacy_helper.erl]: According to **Elvis**:
> Line 181 is too long:         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"iq">>]).

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 2]
```diff
@@ -157,23 +159,14 @@ list_content(<<"deny_not_both_presence_out">>) -> [
         escalus_stanza:privacy_list_item(<<"2">>, <<"deny">>, <<"subscription">>, <<"to">>, [<<"presence-out">>]),
         escalus_stanza:privacy_list_item(<<"3">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"presence-out">>])
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/privacy_helper.erl]: According to **Elvis**:
> Line 160 is too long:         escalus_stanza:privacy_list_item(<<"3">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"presence-out">>]).

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 3]
```diff
@@ -186,18 +179,14 @@ list_content(<<"deny_localhost_iq">>) -> [
...
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"iq">>])
     ];
-list_content(<<"deny_all_iq">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, [<<"iq">>])
-    ];
-list_content(<<"deny_jid_all">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, bob, []),
-        escalus_stanza:privacy_list_jid_item(<<"2">>, <<"deny">>, <<"localhost">>, [])
+list_content(<<"deny_group_presence_out">>) -> [
+        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"presence-out">>])
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/privacy_helper.erl]: According to **Elvis**:
> Line 187 is too long:         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"presence-out">>]).

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 4]
```diff
@@ -114,22 +121,17 @@ is_presence_error(Stanza) ->
...
-    ];
-list_content(<<"allow_bob">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"allow">>, bob, [])
-    ];
-list_content(<<"deny_bob_message">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, bob, [<<"message">>])
+        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, <<"geralt@localhost">>, [])
     ];
 list_content(<<"deny_group_message">>) -> [
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"message">>])
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/privacy_helper.erl]: According to **Elvis**:
> Line 137 is too long:         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"message">>]).

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 5]
```diff
@@ -157,23 +159,14 @@ list_content(<<"deny_not_both_presence_out">>) -> [
...
 list_content(<<"deny_all_presence_in">>) -> [
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, [<<"presence-in">>])
     ];
-list_content(<<"deny_bob_presence_out">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, bob, [<<"presence-out">>])
-    ];
-list_content(<<"deny_group_presence_out">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"presence-out">>])
+list_content(<<"deny_group_presence_in">>) -> [
+        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"presence-in">>])
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/privacy_helper.erl]: According to **Elvis**:
> Line 169 is too long:         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"presence-in">>]).

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 6]
```diff
@@ -157,23 +159,14 @@ list_content(<<"deny_not_both_presence_out">>) -> [
...
         escalus_stanza:privacy_list_item(<<"3">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"presence-out">>])
     ];
-list_content(<<"deny_bob_presence_in">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, bob, [<<"presence-in">>])
-    ];
-list_content(<<"deny_group_presence_in">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"presence-in">>])
-    ];
 list_content(<<"deny_unsubscribed_presence_in">>) -> [
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"presence-in">>])
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/privacy_helper.erl]: According to **Elvis**:
> Line 163 is too long:         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"presence-in">>]).

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 7]
```diff
@@ -157,23 +159,14 @@ list_content(<<"deny_not_both_presence_out">>) -> [
...
-list_content(<<"deny_bob_presence_out">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, bob, [<<"presence-out">>])
-    ];
-list_content(<<"deny_group_presence_out">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"presence-out">>])
+list_content(<<"deny_group_presence_in">>) -> [
+        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"presence-in">>])
     ];
 list_content(<<"deny_unsubscribed_presence_out">>) -> [
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"presence-out">>])
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/privacy_helper.erl]: According to **Elvis**:
> Line 172 is too long:         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"presence-out">>]).

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 8]
```diff
@@ -157,23 +159,14 @@ list_content(<<"deny_not_both_presence_out">>) -> [
         escalus_stanza:privacy_list_item(<<"2">>, <<"deny">>, <<"subscription">>, <<"to">>, [<<"presence-out">>]),
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/privacy_helper.erl]: According to **Elvis**:
> Line 159 is too long:         escalus_stanza:privacy_list_item(<<"2">>, <<"deny">>, <<"subscription">>, <<"to">>, [<<"presence-out">>]),.

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 9]
```diff
@@ -186,18 +179,14 @@ list_content(<<"deny_localhost_iq">>) -> [
     ];
 list_content(<<"deny_group_iq">>) -> [
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"iq">>])
-    ];
+                                     ];
 list_content(<<"deny_unsubscribed_iq">>) -> [
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"iq">>])
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/privacy_helper.erl]: According to **Elvis**:
> Line 184 is too long:         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"iq">>]).

File: test/ejabberd_tests/tests/privacy_SUITE.erl [hunk 10]
```diff
@@ -662,19 +676,21 @@ block_jid_all(Config) ->
 
         %% Alice should NOT receive presence-in from Bob, no err msg
         escalus_client:send(Bob,
-            escalus_stanza:presence_direct(alice, <<"available">>)),
+            escalus_stanza:presence_direct(escalus_utils:get_short_jid(Alice),
+                <<"available">>)),
```
Comment (michalwski) [inline test/ejabberd_tests/tests/privacy_SUITE.erl]: Indentation :)

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 11]
```diff
@@ -114,100 +121,129 @@ is_presence_error(Stanza) ->
...
-list_content(<<"deny_bob">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, bob, [])
-    ];
-list_content(<<"allow_bob">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"allow">>, bob, [])
-    ];
-list_content(<<"deny_bob_message">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, bob, [<<"message">>])
+        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>,
+            <<"geralt@localhost">>, [])
```
Comment (michalwski) [inline test/ejabberd_tests/tests/privacy_helper.erl]: Indentation :)

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 12]
```diff
@@ -114,100 +121,129 @@ is_presence_error(Stanza) ->
...
     ];
 list_content(<<"deny_group_message">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"message">>])
+        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>,
+            <<"ignored">>, [<<"message">>])
```
Comment (michalwski) [inline test/ejabberd_tests/tests/privacy_helper.erl]: Indentation :)

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 13]
```diff
@@ -114,100 +121,129 @@ is_presence_error(Stanza) ->
...
     ];
 list_content(<<"deny_unsubscribed_message">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"message">>])
+        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>,
+            <<"none">>, [<<"message">>])
```
Comment (michalwski) [inline test/ejabberd_tests/tests/privacy_helper.erl]: Indentation :)

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 14]
```diff
@@ -114,100 +121,129 @@ is_presence_error(Stanza) ->
...
     ];
 list_content(<<"deny_all_message">>) -> [
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, [<<"message">>])
     ];
 list_content(<<"deny_all_message_but_subscription_from">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>, <<"from">>, [<<"message">>]),
+        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>,
+            <<"from">>, [<<"message">>]),
```
Comment (michalwski) [inline test/ejabberd_tests/tests/privacy_helper.erl]: Indentation :)

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 15]
```diff
@@ -114,100 +121,129 @@ is_presence_error(Stanza) ->
...
         escalus_stanza:privacy_list_item(<<"2">>, <<"deny">>, [<<"message">>])
     ];
 list_content(<<"deny_all_message_but_subscription_to">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>, <<"to">>, [<<"message">>]),
+        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>,
+            <<"to">>, [<<"message">>]),
```
Comment (michalwski) [inline test/ejabberd_tests/tests/privacy_helper.erl]: Indentation :)

File: test/ejabberd_tests/tests/privacy_helper.erl [hunk 16]
```diff
@@ -114,100 +121,129 @@ is_presence_error(Stanza) ->
...
         escalus_stanza:privacy_list_item(<<"2">>, <<"deny">>, [<<"message">>])
     ];
 list_content(<<"deny_all_message_but_subscription_both">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>, <<"both">>, [<<"message">>]),
+        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>,
+            <<"both">>, [<<"message">>]),
```
Comment (michalwski) [inline test/ejabberd_tests/tests/privacy_helper.erl]: Indentation :)

Comment (michalwski) [inline test/ejabberd_tests/tests/privacy_helper.erl]: It's the last one re indentation but there are similar lines below. Hope you'll find them.

File: test/ejabberd_tests/tests/privacy_SUITE.erl [hunk 17]
```diff
@@ -470,12 +471,15 @@ allow_subscription_to_from_message(Config) ->
 
         %% Alice subscribes to Bob
         escalus_client:send(Alice,
-                            escalus_stanza:presence_direct(bob, <<"subscribe">>)),
+            escalus_stanza:presence_direct(escalus_utils:get_short_jid(Bob),
```
Comment (RoXeon) [inline test/ejabberd_tests/tests/privacy_SUITE.erl]: Most of this code is just rebased from old PR, I just finished "making it work" and code style improvements and making Elvis happy is what I've planed for today :)

Comment (michalwski) [inline test/ejabberd_tests/tests/privacy_SUITE.erl]: Are you sure @RoXeon, you replied to the correct PR? :) Despite that, it sounds like a plan :)

Comment (RoXeon) [inline test/ejabberd_tests/tests/privacy_SUITE.erl]: Oh, sorry. Work PR link :)

**Comment by arkgil** [comment]
@michalwski I've got confused by indentation in this suite. My editor aligns function arguments as you suggested, but  I thought that it is not MongooseIM style compliant 😄  Anyway, I'll take care of that.

**Comment by michalwski** [comment]
I understand that confusion. The SUITE is very old and was written with some freestyle indentation rules.

**Comment by arkgil** [comment]
@michalwski one more thing: should I amend previous commit, since the only changes in it were indentation, or add another one? I feel like with Elvis the amount of indentation commits may grow indefinitely :cry:

**Comment by michalwski** [comment]
Not necessarily :) Let's amend.

I don't think this will change Elvis behaviour as it looks at the PR as a whole.

**Comment by arkgil** [comment]
@michalwski I've submitted the changes

**Pull request merged**